logstash-core/spec/logstash/pipeline_spec.rb

```diff
@@ -247,13 +247,6 @@ class TestPipeline < LogStash::Pipeline
           pipeline = mock_pipeline_from_string(test_config_with_filters)
           pipeline.close
         end
-
-        it "should print the compiled code if config.debug is set to true" do
```

Comment:
Thinking through this, I think we need to keep this giving that we are targeting the 6.1 release. Let's have it just emit `Graph#toString`